settings.gradle

```diff
@@ -10,6 +10,7 @@ include 'contrib'
 
 // Service support
 include 'parsing-utilities'
+include 'permissions'
```

Comment:
Can we add this new top level module description into our [README](https://github.com/NASA-AMMOS/aerie#directory-structure)?

Reply:
Done!